src/main/java/com/wiley/config/AbstractSetConfig.java

```diff
@@ -0,0 +1,12 @@
+package com.wiley.config;
+
+/**
+ * Class for init tests run settings(e.g. browser, grid, timeouts and etc.)
+ * or environment properties(app url, users and etc.)
+ * before start all tests.
+ * Method set() will be invoked in {@link InitConfigListener#onExecutionStart()}.
+ */
+public abstract class AbstractSetConfig {
```

Comment:
it seems that this can become an interface with a noun as a name. Something like CustomizableConfig